src/imdb_rating_predict_model.py

```diff
@@ -0,0 +1,84 @@
+"""Fits a SVR model on the preprocessed data from the IMDB review data set.
+   Saves the model with optimized hyper-parameters, as well as the search result.
+
+Usage:
+  imdb_rating_predict_model.py <train> <out>
+  imdb_rating_predict_model.py (-h | --help)
+
+Options:
+  <train>     Path to the training data file
+  <out>       Path to the directory where output should be written to
+  -h, --help  Display help
+"""
+
+import os
+from pathlib import Path
+
+import joblib
+import numpy as np
+import pandas as pd
+from docopt import docopt
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.model_selection import GridSearchCV
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import (
+    OrdinalEncoder,
+    StandardScaler,
+)
+from sklearn.svm import SVR
+
+numeric_features = ['n_words']
+text_feature = 'Text'
+ordinal_features = ['sentiment']
+target = 'Rating'
+
+
+def main(train, out):
+    # Load data set
+    train_df = pd.read_csv(train)
+    X_train, y_train = train_df.drop(columns=[target, 'Id', 'Author']), train_df[target]
```

Comment:
I am wondering whether it is better to regard 'id' and 'Author' as drop_features is better? Rather than drop them here directly?

Reply:
Yes. I think that's a good idea. I'll add that in.